opentok-angular.js

```diff
@@ -147,6 +147,7 @@ ng.module('opentok', [])
             props.videoContentHint = 'detail';
             publisherVideo = document.createElement('video');
             navigator.mediaDevices.getDisplayMedia({ video: { width, height }, audio: false }).then((stream) => {
+              scope.GDMStream = stream.getTracks()
```

Comment:
personally not a fan of GUM/GDM as var names

Reply:
Let me know what you think of `streamForCanvas`